### PR TITLE
refactor: APT to MOVE

### PR DESCRIPTION
**Pull request by andyjsbell**
## Description

Refactor the name of the native coin from `APT` to `MOVE` and `Aptos Coin` to `Move Coin`

## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Performance improvement
- [x] Refactoring
- [ ] Dependency update
- [ ] Documentation update
- [ ] Tests

## Which Components or Systems Does This Change Impact?
- [ ] Validator Node
- [ ] Full Node (API, Indexer, etc.)
- [x] Move/Aptos Virtual Machine
- [ ] Aptos Framework
- [ ] Aptos CLI/SDK
- [ ] Developer Infrastructure
- [ ] Other (specify)

## How Has This Been Tested?

http://localhost:8080/v1/accounts/0x1/resources and confirming the `0x1::coin::CoinInfo` has the symbol `MOVE`

## Checklist
- [x] I have read and followed the [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I identified and added all stakeholders and component owners affected by this change as reviewers
- [x] I tested both happy and unhappy path of the functionality
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: aptos-move/framework/aptos-framework/sources/aptos_coin.move [hunk 1]
```diff
@@ -41,8 +41,8 @@ module aptos_framework::aptos_coin {
 
         let (burn_cap, freeze_cap, mint_cap) = coin::initialize_with_parallelizable_supply<AptosCoin>(
             aptos_framework,
-            string::utf8(b"Aptos Coin"),
-            string::utf8(b"APT"),
+            string::utf8(b"Move Coin"),
```
Comment (mzabaluev) [inline aptos-move/framework/aptos-framework/sources/aptos_coin.move:44]: May want to rename the file and the Move package to `move_coin` as well, to forestall confusion.
If you do, be sure to find where the file is referenced and update the name there as well.

File: aptos-move/framework/aptos-framework/sources/aptos_coin.spec.move [hunk 2]
```diff
@@ -39,8 +39,8 @@ spec aptos_framework::aptos_coin {
 
         let addr = signer::address_of(aptos_framework);
         aborts_if addr != @aptos_framework;
-        aborts_if !string::spec_internal_check_utf8(b"Aptos Coin");
-        aborts_if !string::spec_internal_check_utf8(b"APT");
+        aborts_if !string::spec_internal_check_utf8(b"Move Coin");
```
Comment (mzabaluev) [inline aptos-move/framework/aptos-framework/sources/aptos_coin.spec.move:42]: Same for this file.

File: aptos-move/framework/aptos-framework/sources/aptos_coin.move [hunk 3]
```diff
@@ -41,8 +41,8 @@ module aptos_framework::aptos_coin {
 
         let (burn_cap, freeze_cap, mint_cap) = coin::initialize_with_parallelizable_supply<AptosCoin>(
```
Comment (mzabaluev) [inline aptos-move/framework/aptos-framework/sources/aptos_coin.move:42]: The data type is still named `AptosCoin`, is it out of scope for this PR?

**Review by mzabaluev — COMMENTED** [review]
The renamings look consistent. I only have a question about the extent we want to rename things at this point. Changing names in the Move code may be less trivial.

**Comment by andyjsbell** [comment]
Right.  This is just skimming the surface and hopefully it can stay this way for some time.  We are effectively just wanting to change the name and ticker for the native coin here.  Refactoring anything else makes me fear breaks in any projects wishing to port across from Aptos.  This needs further thought which I think we all need to sit down and discuss and it could make sense to pull out a subsequent issue to capture this and get this merged.

**Review by mzabaluev — APPROVED** [review]
Looks good for the limited scope of changes as discussed above.

**Comment by andyjsbell** [comment]
thanks.  Is this still the right target for merge? @mzabaluev

**Comment by andyjsbell** [comment]
closing in favour of https://github.com/movementlabsxyz/aptos-core/pull/47

**Pull request closed without merging**